src/plugins/image-plugin.tsx

```diff
@@ -21,10 +22,12 @@ function renderImage(node: Inline, attributes: RenderAttributes, children: React
   const highlightClass = options?.isHighlighted && !options?.isSerializing ? kImageHighlightClass : undefined;
   const classes = mergeClassStrings(highlightClass, attributes.className);
   const src: string = data.get("src");
+  const alt: string = data.get("alt");
   const onLoad = options?.isSerializing ? undefined : options?.onLoad;
   const onClick = options?.isSerializing ? undefined : options?.onClick;
   return (
-    <img className={classes} src={src} onClick={onClick} onLoad={onLoad} {...attributes}/>
+    <img className={classes} src={src} alt={alt} title={alt}
+        onLoad={onLoad} onClick={onClick} {...attributes}/>
```

Comment:
With accessibility becoming a bigger discussion, I'm glad to see the alt and title attributes added.  Should there be some default if the string from the data is empty?

Reply:
Reading through some accessibility guidelines and I get the sense that it might be better to exclude alt/title, as you've done, than trying to slot in some filler text.